Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -384,6 +383,23 @@ public bool DoesntNeedOrHasPower
             return PowerConnection == null || World.Current.PowerNetwork.HasPower(PowerConnection);
         }
     }
+
+    /// <summary>
+    /// Gets the Health of this object.
+    /// </summary>
+    public HealthSystem Health
+    {
+        get
+        {
+            if (health == null)
+            {
+                health = new HealthSystem(-1f, true, false, false, false);
```

Comment:
If health is not defined before being called then we can assume it does not have health. This seems better then setting this in the constructor then changing it in the load data sequence :D